### DISCUSSION TITLE
Linux Mint not supported by install script

### DISCUSSION THREAD

**Discussion by trevormerritt** [SeaQL/FireDBG.for.Rust#42] [Ideas]
Hi!

I like Linux Mint and wanted to use FireDBG.

Turns out the install script doesn't handle mint.

I'm submitting a PR to address this.

I couldn't find any previous discussion regarding OS support or decisions on the topic so I went ahead and added what I needed to make it work on my LMDE6 system, and then also added the wiring for the 'normal' mint installs.

**Comment by billy1624** [comment]
Fixed on

- https://github.com/SeaQL/FireDBG.for.Rust/pull/43

Thanks!! @trevormerritt
